### ISSUE TITLE
Posenet pose detection accuracy issues

### ISSUE THREAD

**Issue by sanzhar0406** [tensorflow/tfjs#867]
I am trying posenet demo to estimate single pose of a person, however even if I run posenet with 1.01 architecture, output stride=8 and image scale factor=1, I am still getting poor keypoint detection. For instance: when I raise my hands up, posenet either does not detect them at all, or confuse right hand with left hand. I tried original posenet demo on Macbook Air, and on Desktop with decent GPU (GTX 1080) and performance on both computers was similar, the only difference was fps. So what is the way to improve keypoint detection accuracy?

**Comment by oveddan** [comment]
The models are pre-trained on the mscoco image dataset, it probably is too generalized to handle these specific cases.  So currently there's no way to fix the higher accuracy issue except with better lighting conditions and camera.

Confusing the right hand with the left hand is an existing bug that happens when flipHorizontal is set to true.     See issue #493 

Let's close this as it is somewhat duplicated in 493

**Comment by rthadur** [comment]
closing this for now.
